scripts/image

```diff
@@ -259,6 +259,12 @@ if [ "$1" = "release" -o "$1" = "mkimage" -o "$1" = "amlpkg" -o "$1" = "noobs" ]
 
   RELEASE_DIR="target/$IMAGE_NAME"
 
```

Comment:
maybe just add a small description here to say what is being done like in the rest of the script?